assignment.js

```diff
@@ -32,6 +32,16 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const baseNumber = 2
+let product
+
+for (let i = 0; i <= 12; i++){
```

Comment:
@justinbudlow great job implementing a `for` loop here. 

An example of an alternative approach to this problem using a while loop [can be found here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)